File: ChallengeBackend/src/main/java/br/com/savedra/challengebackend/model/Invite.java
package br.com.savedra.challengebackend.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Document(collection = "invites")
public class Invite {
    @Id
    private String id;
    private String title;
    private String description;
    private String date;
    private String time;
    private String location;
    private String segment;
}

File: ChallengeCRM/app/src/main/java/br/com/savedra/challengecrm/model/Invite.kt
package br.com.savedra.challengecrm.model

/**
 * Modelo de Convite para o Android.
 */
data class Invite(
  val id: String? = null,
  val title: String? = "",
  val description: String? = "",
  val date: String? = "",
  val time: String? = "",
  val location: String? = "",
  val segment: String? = ""
)
